WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -5,42 +5,32 @@ import Foundation
 /// Notice that the order here matters and it will take
 /// precedence over the backend.
 ///
-/// If the card `isRemote` the `String` should match its
-/// identifier on the backend.
+/// Remote cards should be separately added to RemoteDashboardCard
 enum DashboardCard: String, CaseIterable {
     case quickStart
     case prompts
     case todaysStats = "todays_stats"
```

Comment:
Do we need the `"todays_stats"` here?

Reply:
@leandroalonso This raw value is currently only used for analytics events. And since I'm not sure when this will be released, I didn't want to make any changes to our events before and after this is released. That's why I left this. 
What do you think though? Do you think it's unnecessary?

Reply:
Oh got it, let's def not change analytics values. If we remove it we need to refactor the track to keep the same value as before. If it's easier to just leave that here, let's just leave it.